### PR TITLE
Organize onboarding as a reusable TypeScript scenario

### PR DESCRIPTION
**Pull request by benjaminshafii**
## Summary
Onboarding's environment, user actions, assertions, screenshots, and video now live together under `scenarios/onboarding/`. The workflow is an ordinary async TypeScript function and the video is an ordinary React/Remotion component.

## Why
A single feature previously required following separate world, behavior, spec, and local video files. Colocation gives contributors and agents one place to discover and reuse the journey.

## Scope
- Reuse the existing signup world, testkit channels, and DocShot acceptance loop.
- Add shared field typing with deterministic grapheme cadence, one focus per field, final-value verification, and masked password input.
- Extract the existing browser frame and timestamp-based recording playback into `packages/presentation`.
- Include scenario tests in the existing CLI, Vitest projects, and boundary/channel checks. Keep secret-bearing PR CI on its existing legacy-spec-only matrix; scenario execution is explicit. Preserve existing spec paths and explicit live-test opt-ins.
- Keep the recorded onboarding full screen and show download progress from browser events. The empty-chat ending is an illustration.

## Testing
Head: `f8332cbc4d1abe4912515065890852d524348a17`.

- `node --test --test-concurrency=1 evals/bin/evals.test.mjs evals/scripts/spec-boundary-ratchet.test.mjs evals/scripts/spec-channel-ratchet.test.mjs evals/packages/behaviors/test/typing.test.ts packages/presentation/test/recording.test.ts`: 30 passed, 0 failed, 0 skipped.
- `pnpm --dir scenarios typecheck`: passed.
- `pnpm --dir evals run typecheck:spec-layer`: passed.
- `env -u OPENWORK_EVAL_FILM_DIR -u OPENWORK_EVAL_DEN_API_URL -u OPENWORK_EVAL_DEN_WEB_URL OPENWORK_EVAL_REF=f8332cbc4d1abe4912515065890852d524348a17 pnpm evals:e2e onboarding`: **Passed**, 1 passed / 0 failed / 0 skipped, 330.88 seconds. Printed placement: `daytona (daytona CLI authenticated)`. Both provisioned sandboxes were deleted.
- Six observable assertions passed: account creation, one Studio workspace, exactly two member invitations and outbox deliveries, exactly two unconnected personal tool configurations, completed installer bytes/GUID, and accepted DocShot capture.
- Two supporting screenshots remain explicitly unvalidated by a vision model; they do not determine the assertion verdict.
- Demonstration recording from `cf7538a79` (presentation code unchanged): 256 captured frames, 38.871 seconds, installer 208,556,810 / 208,556,810 bytes. `pnpm --dir scenarios onboarding:render <capture-directory>` passed. The 46.955-second, 1920×1200 H.264 output decoded fully without errors; invitation, tool, completed-download, and empty-chat frames were visually inspected.

The channel/boundary ratchets fail with the same signatures on clean `dev` at `7e396788e`. Dependency checking under Node 24.12.0 reproduces the same 18 existing violations on that control checkout; no scenario violations. The default local Node 25 is unsupported by dependency-cruiser, so both comparisons used Node 24.

## CI status
Required `openwork-tests-required` CI passed on the final head. Warden completed with zero findings and approved the final head. Local legacy ratchet and dependency failures are documented above with a clean-dev comparison.

## Manual verification
From the repository root after installing the root and evals workspaces:

```sh
node evals/bin/evals.mjs --list
OPENWORK_EVAL_REF=f8332cbc4d1abe4912515065890852d524348a17 OPENWORK_EVAL_FILM_DIR="$PWD/evals/results/onboarding-review" pnpm evals:e2e onboarding
pnpm --dir scenarios onboarding:render "$PWD/evals/results/onboarding-review"
```

Choose a fresh recording directory for each run. Accounts and invitations use isolated synthetic fixtures and the development outbox. The installer download requires public GitHub access.

## Evidence
The completed head run is published in the sticky test-evidence comment. The rendered recording is attached in the demo comment.

## Risk
This is an initial onboarding migration. Legacy world provisioning and DocShot scene registration remain in place. Remotion adds renderer dependencies to the evals/scenarios workspace. The rendered empty chat is not evidence of desktop installation or provider authorization.

## Rollback
Revert this PR; the legacy spec runner and product onboarding remain available.

### CODE REVIEW

**Comment by benjaminshafii** [comment]
## Test evidence — signup-invite-two-teammates-add-tools-and-download-the-desktop-app — ✅ passed

SHA cf7538a794c959c9e7122fcbc7acd5ecc8bc0a9e · engine v1

✅ **0/2 screenshots passed · 6 assertions** · 6 expectations passed · 0 failed · 0 pending

**[world]** den(daytona) · web(signed out)
**[user]** see(text=Good work starts here., timeoutMs=90000) · type(label=Email, "<email>", replace) · label=Next · click(label=Next) · type(label=Name, "Alex", replace) · type(label=Password, <redacted>, replace) · label=Sign up · click(label=Sign up) · see(text=Make it yours., timeoutMs=90000)
**[probe]** api(GET /v1/me/orgs)
**[user]** text=On my own · click(text=On my own) · type(label=Organization name, "Studio", replace) · label=Continue · click(label=Continue) · see(text=Bring your people., timeoutMs=90000)
**[probe]** api(GET /v1/me/orgs)
**[user]** type(label=Teammate email 1, "<email>", replace) · type(label=Teammate email 2, "<email>", replace) · label=Send invitations · click(label=Send invitations) · see(text=2 invitations sent., timeoutMs=90000)
**[probe]** api(GET /v1/org) · api(GET /v1/dev/emails?template=organizationInvite)
**[user]** label=Continue · click(label=Continue) · see(text=Give your team a head start., timeoutMs=90000) · label=Add Notion · click(label=Add Notion) · label=Add Linear · click(label=Add Linear) · label=Add to team · click(label=Add to team) · see(text=Added to team, timeoutMs=90000)
**[probe]** api(GET /v1/mcp-connections?scope=manageable)
**[user]** label=Continue · click(label=Continue) · see(text=Download for Linux, timeoutMs=90000) · text=Download for Linux · click(text=Download for Linux)
**[probe]** eventually(installer completed)
**steps** 1 ✅ Enter email (4.1s) · 2 ✅ Continue to account details (0.4s) · 3 ✅ Enter name (2.7s) · 4 ✅ Enter password (7.5s) · 5 ✅ Create account (1.7s) · 6 ✅ Choose personal workspace (0.4s) · 7 ✅ Name workspace (2.1s) · 8 ✅ Create workspace (0.9s) · 9 ✅ Enter teammate 1 (4.2s) · 10 ✅ Enter teammate 2 (6.4s) · 11 ✅ Send invitations (0.9s) · 12 ✅ Continue to tools (1.1s) · 13 ✅ Select Notion (0.8s) · 14 ✅ Select Linear (0.6s) · 15 ✅ Add selected tools (0.7s) · 16 ✅ Continue to download (0.6s) · 17 ✅ Download OpenWork (0.4s) · 18 ✅ Download completes (1.5s)
**verdict** passed · 7 user observations (see ×7) · 6 probes · steps 18/18

### ℹ️ ASSERTION — 1. Account created through signup

No organization existed before the workspace step

- ✅ **PASS** Account created through signup — No organization existed before the workspace step

### ℹ️ ASSERTION — 2. One Studio workspace created

[{&quot;id&quot;:&quot;org_01m1t7he77ewg8j0gb2fsp1tzy&quot;,&quot;name&quot;:&quot;Studio&quot;,&quot;slug&quot;:&quot;org_01m1t7he77ewg8j0gb2fsp1tzy&quot;,&quot;logo&quot;:null,&quot;allowedEmailDomains&quot;:null,&quot;metadata&quot;:&quot;{\&quot;limits\&quot;:{\&quot;members\&quot;:5,\&quot;workers\&quot;:1}}&quot;,&quot;role&quot;:&quot;owner&quot;,&quot;orgMemberId&quot;:&quot;om_01m1t7he78efna0bdykf7t7cgg&quot;,&quot;membershipId&quot;:&quot;om_01m1t7he78efna0bdykf7t7cgg&quot;,&quot;memberCount&quot;:1,&quot;createdAt&quot;:&quot;2026-09-06T02:07:43.848Z&quot;,&quot;updatedAt&quot;:&quot;2026-09-06T02:07:43.848Z&quot;,&quot;isActive&quot;:true}]

- ✅ **PASS** One Studio workspace created — [{&quot;id&quot;:&quot;org_01m1t7he77ewg8j0gb2fsp1tzy&quot;,&quot;name&quot;:&quot;Studio&quot;,&quot;slug&quot;:&quot;org_01m1t7he77ewg8j0gb2fsp1tzy&quot;,&quot;logo&quot;:null,&quot;allowedEmailDomains&quot;:null,&quot;metadata&quot;:&quot;{\&quot;limits\&quot;:{\&quot;members\&quot;:5,\&quot;workers\&quot;:1}}&quot;,&quot;role&quot;:&quot;owner&quot;,&quot;orgMemberId&quot;:&quot;om_01m1t7he78efna0bdykf7t7cgg&quot;,&quot;membershipId&quot;:&quot;om_01m1t7he78efna0bdykf7t7cgg&quot;,&quot;memberCount&quot;:1,&quot;createdAt&quot;:&quot;2026-09-06T02:07:43.848Z&quot;,&quot;updatedAt&quot;:&quot;2026-09-06T02:07:43.848Z&quot;,&quot;isActive&quot;:true}]

### ℹ️ ASSERTION — 3. Two teammates invited once as members

{&quot;invitations&quot;:[{&quot;id&quot;:&quot;inv_01m1t7hspfe27stg5dy1n2nagd&quot;,&quot;email&quot;:&quot;casey@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:07:55.599Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:07:55.599Z&quot;,&quot;inviteToken&quot;:&quot;TP2VYcwkvGqYcu9WDW0jY&quot;},{&quot;id&quot;:&quot;inv_01m1t7hssee3e8183qtxahwrf2&quot;,&quot;email&quot;:&quot;jordan@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:07:55.694Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:07:55.694Z&quot;,&quot;inviteToken&quot;:&quot;uS8NDjubKZdyXF4Dn5gNu&quot;}],&quot;delivery&quot;:&quot;development outbox&quot;}

- ✅ **PASS** Two teammates invited once as members — {&quot;invitations&quot;:[{&quot;id&quot;:&quot;inv_01m1t7hspfe27stg5dy1n2nagd&quot;,&quot;email&quot;:&quot;casey@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:07:55.599Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:07:55.599Z&quot;,&quot;inviteToken&quot;:&quot;TP2VYcwkvGqYcu9WDW0jY&quot;},{&quot;id&quot;:&quot;inv_01m1t7hssee3e8183qtxahwrf2&quot;,&quot;email&quot;:&quot;jordan@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:07:55.694Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:07:55.694Z&quot;,&quot;inviteToken&quot;:&quot;uS8NDjubKZdyXF4Dn5gNu&quot;}],&quot;delivery&quot;:&quot;development outbox&quot;}

### ℹ️ ASSERTION — 4. Notion and Linear added for the workspace

Exactly two configurations; neither personal account is authorized

- ✅ **PASS** Notion and Linear added for the workspace — Exactly two configurations; neither personal account is authorized

### ℹ️ ASSERTION — 5. Desktop installer downloaded completely

{&quot;guid&quot;:&quot;72a4ef37-6aa8-4da6-91fc-652235749d96&quot;,&quot;totalBytes&quot;:208556810,&quot;receivedBytes&quot;:208556810,&quot;state&quot;:&quot;completed&quot;,&quot;filePath&quot;:&quot;/tmp/openwork-film-downloads-ef57eb15-2a85-456a-ac39-9b0f2c07f657/openwork-linux-x86_64-0.18.42.AppImage&quot;,&quot;event&quot;:&quot;Browser.downloadProgress&quot;,&quot;receivedAt&quot;:1788660482924}

- ✅ **PASS** Desktop installer downloaded completely — {&quot;guid&quot;:&quot;72a4ef37-6aa8-4da6-91fc-652235749d96&quot;,&quot;totalBytes&quot;:208556810,&quot;receivedBytes&quot;:208556810,&quot;state&quot;:&quot;completed&quot;,&quot;filePath&quot;:&quot;/tmp/openwork-film-downloads-ef57eb15-2a85-456a-ac39-9b0f2c07f657/openwork-linux-x86_64-0.18.42.AppImage&quot;,&quot;event&quot;:&quot;Browser.downloadProgress&quot;,&quot;receivedAt&quot;:1788660482924}

### ℹ️ ASSERTION — 6. DocShot captures the settled download page

{&quot;file&quot;:&quot;screenshots/download.png&quot;,&quot;bytes&quot;:149073}

- ✅ **PASS** DocShot captures the settled download page — {&quot;file&quot;:&quot;screenshots/download.png&quot;,&quot;bytes&quot;:149073}

### ⚪ UNVALIDATED — 7. signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 6

- ⚪ **UNVALIDATED** — no visual expectations recorded.

signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 6

### ⚪ UNVALIDATED — 8. signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 7

- ⚪ **UNVALIDATED** — no visual expectations recorded.

signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 7

---
_Test run created 2026-09-06T02:04:00.848Z · Source: `evals/results/test-runs/2026-09-06T02-04-00-848Z-signup-invite-two-teammates-add-tools-and-download-the-desktop-app/test-run.json` · Repro: `pnpm --dir evals artifacts:publish -- --pr 4546 --test-run 2026-09-06T02-04-00-848Z-signup-invite-two-teammates-add-tools-and-download-the-desktop-app`_

**Comment by benjaminshafii** [comment]
Fresh onboarding recording from PR head `cf7538a794c959c9e7122fcbc7acd5ecc8bc0a9e`: actual signup, two member invitations, Notion and Linear selection, and a completed 208,556,810-byte installer download. Field input uses the shared paced-typing behavior. The browser remains full screen.

The final empty-chat opening is an illustration. This video does not claim that desktop installation or provider authorization was tested.

Rendered using the committed `scenarios/onboarding/render.mjs`; full video decoding and four key frames were checked.

https://github.com/user-attachments/assets/61ac6d39-da10-4998-b14c-fc6443bb5795

File: evals/bin/test-files.mjs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          .includes("node_modules"),
+    )
+    .map((entry) => join(entry.parentPath, entry.name))
+    .sort();
+}
+
+export function journeyFiles(pattern = /(?:^|\.)e2e\.test\.ts$/) {
+  return [
+    ...filesUnder(specsRoot, pattern),
+    ...filesUnder(scenariosRoot, pattern),
```
[the remark this answers is not in the document]
Comment (benjaminshafii) [reply evals/bin/test-files.mjs:27]: Removed the PR changes to .github/workflows/daytona-e2e.yml in f8332cbc4. The privileged workflow retains its original legacy-spec-only matrix and does not invoke scenario discovery. CLI/Vitest discovery remains available for explicitly initiated scenario runs. No new scenario path is selected by the secret-bearing PR workflow.

File: evals/specs/signup-workspace-intent.e2e.test.ts [hunk 2]
```diff
@@ -131,6 +131,23 @@ test("signup distinguishes joining, personal work, and restricted team setup wit
     expect(await probe.eval("document.querySelectorAll('[data-testid=download-openwork-card] details a[href]').length")).toBe(8);
     await user.click({ text: "Other platforms and versions" });
     await user.looks(["The final setup screen shows a clear desktop download and model setup path in the same restrained black-and-white design"]);
+    if (world.film) {
+      await step("the real Linux desktop download completes", async () => {
+        await user.click({ role: "link", text: "Download for Linux" });
```
[the remark this answers is not in the document]
Comment (benjaminshafii) [reply evals/specs/signup-workspace-intent.e2e.test.ts]: Removed the optional download assertion from the legacy signup spec in f8332cbc4. Download completion is owned by scenarios/onboarding/e2e.test.ts, whose world always creates capture using an isolated temporary directory even without OPENWORK_EVAL_FILM_DIR. The scenario unconditionally verifies the installer filename, GUID, received/total bytes, and absence of cancellation. A fresh cold run without the film environment variable is underway on this head.

**Comment by benjaminshafii** [comment]
## Test evidence — signup-invite-two-teammates-add-tools-and-download-the-desktop-app — ✅ passed

SHA f8332cbc4d1abe4912515065890852d524348a17 · engine v1

✅ **0/2 screenshots passed · 6 assertions** · 6 expectations passed · 0 failed · 0 pending

**[world]** den(daytona) · web(signed out)
**[user]** see(text=Good work starts here., timeoutMs=90000) · type(label=Email, "<email>", replace) · label=Next · click(label=Next) · type(label=Name, "Alex", replace) · type(label=Password, <redacted>, replace) · label=Sign up · click(label=Sign up) · see(text=Make it yours., timeoutMs=90000)
**[probe]** api(GET /v1/me/orgs)
**[user]** text=On my own · click(text=On my own) · type(label=Organization name, "Studio", replace) · label=Continue · click(label=Continue) · see(text=Bring your people., timeoutMs=90000)
**[probe]** api(GET /v1/me/orgs)
**[user]** type(label=Teammate email 1, "<email>", replace) · type(label=Teammate email 2, "<email>", replace) · label=Send invitations · click(label=Send invitations) · see(text=2 invitations sent., timeoutMs=90000)
**[probe]** api(GET /v1/org) · api(GET /v1/dev/emails?template=organizationInvite)
**[user]** label=Continue · click(label=Continue) · see(text=Give your team a head start., timeoutMs=90000) · label=Add Notion · click(label=Add Notion) · label=Add Linear · click(label=Add Linear) · label=Add to team · click(label=Add to team) · see(text=Added to team, timeoutMs=90000)
**[probe]** api(GET /v1/mcp-connections?scope=manageable)
**[user]** label=Continue · click(label=Continue) · see(text=Download for Linux, timeoutMs=90000) · text=Download for Linux · click(text=Download for Linux)
**[probe]** eventually(installer completed)
**steps** 1 ✅ Enter email (6.1s) · 2 ✅ Continue to account details (0.8s) · 3 ✅ Enter name (2.7s) · 4 ✅ Enter password (6.7s) · 5 ✅ Create account (2.5s) · 6 ✅ Choose personal workspace (0.5s) · 7 ✅ Name workspace (2.7s) · 8 ✅ Create workspace (1.3s) · 9 ✅ Enter teammate 1 (5.8s) · 10 ✅ Enter teammate 2 (6.7s) · 11 ✅ Send invitations (1.3s) · 12 ✅ Continue to tools (1.3s) · 13 ✅ Select Notion (0.8s) · 14 ✅ Select Linear (0.8s) · 15 ✅ Add selected tools (1.0s) · 16 ✅ Continue to download (1.1s) · 17 ✅ Download OpenWork (0.9s) · 18 ✅ Download completes (59.7s)
**verdict** passed · 7 user observations (see ×7) · 6 probes · steps 18/18

### ℹ️ ASSERTION — 1. Account created through signup

No organization existed before the workspace step

- ✅ **PASS** Account created through signup — No organization existed before the workspace step

### ℹ️ ASSERTION — 2. One Studio workspace created

[{&quot;id&quot;:&quot;org_01m1t9xtfae9gt54zd1cb1q5ct&quot;,&quot;name&quot;:&quot;Studio&quot;,&quot;slug&quot;:&quot;org_01m1t9xtfae9gt54zd1cb1q5ct&quot;,&quot;logo&quot;:null,&quot;allowedEmailDomains&quot;:null,&quot;metadata&quot;:&quot;{\&quot;limits\&quot;:{\&quot;members\&quot;:5,\&quot;workers\&quot;:1}}&quot;,&quot;role&quot;:&quot;owner&quot;,&quot;orgMemberId&quot;:&quot;om_01m1t9xtfbe84vt66axkbsw62s&quot;,&quot;membershipId&quot;:&quot;om_01m1t9xtfbe84vt66axkbsw62s&quot;,&quot;memberCount&quot;:1,&quot;createdAt&quot;:&quot;2026-09-06T02:49:26.763Z&quot;,&quot;updatedAt&quot;:&quot;2026-09-06T02:49:26.763Z&quot;,&quot;isActive&quot;:true}]

- ✅ **PASS** One Studio workspace created — [{&quot;id&quot;:&quot;org_01m1t9xtfae9gt54zd1cb1q5ct&quot;,&quot;name&quot;:&quot;Studio&quot;,&quot;slug&quot;:&quot;org_01m1t9xtfae9gt54zd1cb1q5ct&quot;,&quot;logo&quot;:null,&quot;allowedEmailDomains&quot;:null,&quot;metadata&quot;:&quot;{\&quot;limits\&quot;:{\&quot;members\&quot;:5,\&quot;workers\&quot;:1}}&quot;,&quot;role&quot;:&quot;owner&quot;,&quot;orgMemberId&quot;:&quot;om_01m1t9xtfbe84vt66axkbsw62s&quot;,&quot;membershipId&quot;:&quot;om_01m1t9xtfbe84vt66axkbsw62s&quot;,&quot;memberCount&quot;:1,&quot;createdAt&quot;:&quot;2026-09-06T02:49:26.763Z&quot;,&quot;updatedAt&quot;:&quot;2026-09-06T02:49:26.763Z&quot;,&quot;isActive&quot;:true}]

### ℹ️ ASSERTION — 3. Two teammates invited once as members

{&quot;invitations&quot;:[{&quot;id&quot;:&quot;inv_01m1t9y88ge6b989340zc416nv&quot;,&quot;email&quot;:&quot;casey@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:49:40.880Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:49:40.881Z&quot;,&quot;inviteToken&quot;:&quot;wJtA2gEayOQieNuCuaj9p&quot;},{&quot;id&quot;:&quot;inv_01m1t9y8che8nv5tqmpaye0bvp&quot;,&quot;email&quot;:&quot;jordan@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:49:41.009Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:49:41.010Z&quot;,&quot;inviteToken&quot;:&quot;7oAxNpB15sBep2z1dLoP7&quot;}],&quot;delivery&quot;:&quot;development outbox&quot;}

- ✅ **PASS** Two teammates invited once as members — {&quot;invitations&quot;:[{&quot;id&quot;:&quot;inv_01m1t9y88ge6b989340zc416nv&quot;,&quot;email&quot;:&quot;casey@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:49:40.880Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:49:40.881Z&quot;,&quot;inviteToken&quot;:&quot;wJtA2gEayOQieNuCuaj9p&quot;},{&quot;id&quot;:&quot;inv_01m1t9y8che8nv5tqmpaye0bvp&quot;,&quot;email&quot;:&quot;jordan@openwork.test&quot;,&quot;role&quot;:&quot;member&quot;,&quot;status&quot;:&quot;pending&quot;,&quot;expiresAt&quot;:&quot;2026-09-13T02:49:41.009Z&quot;,&quot;createdAt&quot;:&quot;2026-09-06T02:49:41.010Z&quot;,&quot;inviteToken&quot;:&quot;7oAxNpB15sBep2z1dLoP7&quot;}],&quot;delivery&quot;:&quot;development outbox&quot;}

### ℹ️ ASSERTION — 4. Notion and Linear added for the workspace

Exactly two configurations; neither personal account is authorized

- ✅ **PASS** Notion and Linear added for the workspace — Exactly two configurations; neither personal account is authorized

### ℹ️ ASSERTION — 5. Desktop installer downloaded completely

{&quot;guid&quot;:&quot;23f376cf-4f34-477b-a85b-8720c3824ab5&quot;,&quot;totalBytes&quot;:208556810,&quot;receivedBytes&quot;:208556810,&quot;state&quot;:&quot;completed&quot;,&quot;filePath&quot;:&quot;/tmp/openwork-film-downloads-eb842e1c-3782-4daf-901e-633c4247a4e8/openwork-linux-x86_64-0.18.42.AppImage&quot;,&quot;event&quot;:&quot;Browser.downloadProgress&quot;,&quot;receivedAt&quot;:1788663047553}

- ✅ **PASS** Desktop installer downloaded completely — {&quot;guid&quot;:&quot;23f376cf-4f34-477b-a85b-8720c3824ab5&quot;,&quot;totalBytes&quot;:208556810,&quot;receivedBytes&quot;:208556810,&quot;state&quot;:&quot;completed&quot;,&quot;filePath&quot;:&quot;/tmp/openwork-film-downloads-eb842e1c-3782-4daf-901e-633c4247a4e8/openwork-linux-x86_64-0.18.42.AppImage&quot;,&quot;event&quot;:&quot;Browser.downloadProgress&quot;,&quot;receivedAt&quot;:1788663047553}

### ℹ️ ASSERTION — 6. DocShot captures the settled download page

{&quot;file&quot;:&quot;screenshots/download.png&quot;,&quot;bytes&quot;:149073}

- ✅ **PASS** DocShot captures the settled download page — {&quot;file&quot;:&quot;screenshots/download.png&quot;,&quot;bytes&quot;:149073}

### ⚪ UNVALIDATED — 7. signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 6

- ⚪ **UNVALIDATED** — no visual expectations recorded.

signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 6

### ⚪ UNVALIDATED — 8. signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 7

- ⚪ **UNVALIDATED** — no visual expectations recorded.

signup-invite-two-teammates-add-tools-and-download-the-desktop-app artifact 7

---
_Test run created 2026-09-06T02:45:24.131Z · Source: `evals/results/test-runs/2026-09-06T02-45-24-131Z-signup-invite-two-teammates-add-tools-and-download-the-desktop-app/test-run.json` · Repro: `pnpm --dir evals artifacts:publish -- --pr 4546 --test-run 2026-09-06T02-45-24-131Z-signup-invite-two-teammates-add-tools-and-download-the-desktop-app`_

File: evals/specs/signup-workspace-intent.e2e.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (benjaminshafii) [reply evals/specs/signup-workspace-intent.e2e.test.ts]: The final-head cold Daytona run passed with OPENWORK_EVAL_FILM_DIR unset: 1 passed, 0 failed, 0 skipped. All six assertions passed, including the mandatory completed installer witness (208,556,810 / 208,556,810 bytes). Current-head evidence is published at https://github.com/different-ai/openwork/pull/4546#issuecomment-5556462840.
